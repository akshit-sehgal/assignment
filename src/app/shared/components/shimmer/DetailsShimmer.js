import React from 'react';
import Shimmer from './Shimmer';
import './detailsshimmer.scss';

const DetailsShimmer = () => {
    return (
        <div className="details-shimmer">
            <div className="ds__left">
                <Shimmer showLabelsHolder={false} />
            </div>
            <div className="ds__right">
                <Shimmer showImageHolder={false} count={2} />
            </div>
        </div>
    )
}
export default DetailsShimmer;