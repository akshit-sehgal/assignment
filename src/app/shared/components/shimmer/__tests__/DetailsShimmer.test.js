import React from 'react';
import { shallow } from 'enzyme';
import DetailsShimmer from '../DetailsShimmer';

describe('Testing <DetailsShimmer/>', () => {
    it('should render the default view', () => {
        const wrapper = shallow(<DetailsShimmer />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('.ds__left').length).toEqual(1);
        expect(wrapper.find('.ds__right').length).toEqual(1);
    });
});