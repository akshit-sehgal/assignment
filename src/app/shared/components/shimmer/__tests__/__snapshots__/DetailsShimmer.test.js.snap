// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <DetailsShimmer/> should render the default view 1`] = `
<div
  className="details-shimmer"
>
  <div
    className="ds__left"
  >
    <Shimmer
      count={1}
      showImageHolder={true}
      showLabelsHolder={false}
    />
  </div>
  <div
    className="ds__right"
  >
    <Shimmer
      count={2}
      showImageHolder={false}
      showLabelsHolder={true}
    />
  </div>
</div>
`;
